### DISCUSSION TITLE
Opening floating documentation window in new tab/window.

### DISCUSSION THREAD

**Discussion by iukej** [neoclide/coc.nvim#2933] [Q&A] [answered]
How would it be possible to split the documentation floating window into a new tab or window?

I've seen recommendations for using control-w, but this exits the floating window.

**Comment by chemzqm** [comment, marked as answer]
It's not possible.

**Reply by iukej** [reply]
I see. How would you recommend going about getting this information and putting it into a new window? I find it so useful to have documentation visible while writing.

**Reply by chemzqm** [reply]
Try `<C-w>p` and `<C-w>T` after open float window on neovim.
